SocketIOClientSwift/WebSocket.swift

```diff
@@ -378,21 +378,17 @@ public class WebSocket : NSObject, NSStreamDelegate {
         }
         if totalSize > 0 {
             if validateResponse(buffer, bufferLen: totalSize) {
-                dispatch_async(queue, { [weak self] in
```

Comment:
This strong reference is only kept for a short period of time (while this block is invoked). This **won't** cause a retain cycle. I'll explain more separately though.